provider/pkg/provider/provider.go

```diff
@@ -677,8 +677,8 @@ func (k *kubeProvider) Configure(_ context.Context, req *pulumirpc.ConfigureRequ
 		config, err := kubeconfig.ClientConfig()
 		if err != nil {
 			k.clusterUnreachable = true
-			k.clusterUnreachableReason = fmt.Sprintf(
-				"unable to load Kubernetes client configuration from kubeconfig file: %v", err)
+			k.clusterUnreachableReason = fmt.Sprintf("unable to locate kubeconfig. Make sure you have: \n\n" +
+			" \t • Setup the provider as per https://pulumi.io/install/kubernetes.html \n\n")
```

Comment:
1. Need to gofmt file
2. https://www.pulumi.com/registry/packages/kubernetes/installation-configuration/ might be a better link. The current one is a `pulumi.io` link

Reply:
Thanks for the review! I'm going to close this and tweak a bit per convo with Paul, but wanted to point out that the URL is a redirect to the one you suggest, following a pattern we are using in aws-native, opinions @stack72 ?

Reply:
Thinking about it more it's smart to use the shorter redirect URL not only because it looks better but because it allows the registry to adjust URLs without falling out of sync with individual providers, instead maintaining one pulumi.io redirect list.

Reply:
I updated this PR @lblackstone would you mind having another look when you get a chance?